applications/ortege.md

```diff
@@ -0,0 +1,267 @@
+
+To provide transparency and traceability, we have deployed an explorer that can track the entire lifecycle of a message. This tool allows users to monitor the status of their transactions and provides valuable insights into the operation of the protocol. 
+
+Lastly, we have integrated WalletConnect 2 to enable the remote control of an interchain account on another chain. This feature enhances the user experience by allowing users to manage their assets across different chains from a single interface. 
+
+In summary, while there is still work to be done, we are proud of the progress we have made so far and are excited about the potential of Ortege to revolutionize blockchain interoperability. 
+
+## Development Roadmap :nut_and_bolt: 
+ ### Overview 
+- **Total Estimated Duration:** 5 months 
+
+- **Full-Time Equivalent (FTE):**  4 FTE (Milestones will be worked on concurrently) 
+
+- **Total Costs:** $95,000 
+
+  
+
+### Milestone 1 Ink! Smart Contracts 
+
+We will begin by developing smart contracts in the ink! language. These contracts will form the backbone of our protocol, enabling the core functionality of cross-chain communication. We will leverage the design principles and specifications provided by Hyperlane to ensure our contracts are robust, secure, and efficient. This will involve creating contracts for managing the protocol's state, handling message routing, and facilitating the transfer of value across chains. 
+ 
+- **Estimated duration:** 4 months 
+- **FTE:**  2.5 
+- **Costs:** 60,000 USD 
+
+| Number | Deliverable | Specification | 
+| ------ | ----------- | ------------- | 
+| **0a.** | License | Apache 2.0 | 
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can query, call or post an interchain message. | 
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. | 
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. | 
+| **0e.** | Article | We will publish an **article**/workshop that explains [...] (what was done/achieved as part of the grant). | 
+| **1.** | Ink! Contract: MerkleTree | We will create an ink! MerkleTree implementation and will submit to OpenBrush for inclusion in their standards | 
+| **2.** | Ink! Contract: Interchain security module | The ISM contract will provide modular security for the Ortege system, accommodating use-case-specific security requirements. | 
```

Comment:
Can you elaborate on what these security requirements look like and how they are added/modified/maintained? Hyperlane seems to provide an `Interface` and several default implementations, will this be the case here? If so, could you add this information?

Reply:
We will provide the Interface and as part of this scope of work, the multisig ISM. Will update the application to reflect this